package.json

```diff
@@ -3,14 +3,17 @@
   "version": "0.1.0",
```

Comment:
- [ ] The MIT license link in your README file is broken. I think this is happening because the link is referencing `LICENSE.md` while your file is named `LICENSE`. Please fix the link using the correct name. Having a license in a README file makes your repository more professional. 🤓️

![image](https://github.com/abenezer-addisu/React-Bookstore/assets/17243325/84486b68-87a6-48a9-b1d5-342b6195335c)